packages/venia-ui/lib/components/MyAccount/myAccount.js

```diff
@@ -0,0 +1,57 @@
+import React, { useCallback } from 'react';
+import { Archive as HistoryIcon, LogOut as SignOutIcon } from 'react-feather';
+import { func, shape, string } from 'prop-types';
+
+import { mergeClasses } from '../../classify';
+import AccountLink from './accountLink';
+import defaultClasses from './myAccount.css';
+
+const MyAccount = props => {
+    const { signOut, user } = props;
+    const { email, firstname, lastname } = user;
+    const name = `${firstname} ${lastname}`.trim() || 'My Account';
```

Comment:
These string defaults could be made constants.

Reply:
This has been addressed.